firestore.rules

```diff
@@ -223,11 +235,14 @@ service cloud.firestore {
     }
 
     match /members/{membersId} {
-      allow read, create, update: if isAdmin();
+      allow read, delete: if isAdmin();
+      allow create: if documentFieldsCheckOut(['timestamp', 'course', 'email', 'paymentStatus', 'enrollmentNumber', 'facultyNumber', 'mobile', 'name', 'registrationNumber'], []) && memberFieldsTypeValidate(request.resource.data);
+      allow update: if isAdmin() && editOnlyChangesField(['timestamp', 'course', 'email', 'paymentStatus', 'enrollmentNumber', 'facultyNumber', 'mobile', 'name', 'registrationNumber']) && memberFieldsTypeValidate(request.resource.data);
     }
 
     match /facultyNumbers/{facultyNumbersId} {
-      allow read, create, update: if isAdmin();
+      allow read, write;
```

Comment:
`write`  consists of all three `create`, `update` and `delete`, so if you once allow write to everyone then it'll not check next line i.e `allow delete, update: if isAdmin();` 
Please fix.

Reply:
@Rishabhltfb, this change is outdated. Please review the latest changes :sweat_smile: